tools/ci-build/scripts/check-server-e2e-test

```diff
@@ -7,4 +7,6 @@ set -eux
 cd smithy-rs/rust-runtime/aws-smithy-http-server/examples
 
 make
-cargo test
+make clippy
```

Comment:
It should suffice to just have:

```
make test clippy
```

in both `Makefiles`.

Reply:
So you want me to edit the `Makefile`s? Or you mean in the `check-server-(python-)e2e-test` files to just condense what I have into `make test clippy` (I am assuming the latter, just want to make sure).

Reply:
The latter.